### PR TITLE
Add copy ingest processor

### PR DESCRIPTION
**Pull request by gaobinlong**
### Description
This PR documents the usage of the new added copy ingest processor.

### Issues Resolved
Fixes https://github.com/opensearch-project/documentation-website/issues/6147

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by gaobinlong** [comment]
Hi @vagimeli, please help to review this PR, thanks!

**Review by vagimeli — APPROVED** [review]

**Comment by vagimeli** [comment]
> Hi @vagimeli, please help to review this PR, thanks!

@gaobinlong Thank you for putting up the PR. Please see my edits for your approval. I'll move this into editorial review.

File: _ingest-pipelines/processors/copy.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  ]
+}
+```
+
+**Step 3: Ingest a document**
+
+The following query ingests a document into an index named `testindex1`:
+
+```json
+PPUT testindex1/_doc/1?pipeline=copy_object
```
Comment (vagimeli) [inline _ingest-pipelines/processors/copy.md:134]:
```suggestion
PUT testindex1/_doc/1?pipeline=copy_object
```

**Review by vagimeli — APPROVED** [review]

File: _ingest-pipelines/processors/copy.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+`ignore_missing`  | Optional  | Specifies whether the processor should ignore documents that do not contain the specified `source_field`. If set to `true`, the processor does not modify the document if the `source_field` does not exist or is `null`. Default is `false`. |
+`override_target`  | Optional  | Specifies whether the processor should override the `target_field` if it already exists in the document. If set to `true`, the processor overrides the value of `target_field` if it already exists. Default is `false`. |
+`remove_source`  | Optional  | Specifies whether the processor should remove the `source_field` after it's been copied. If set to `true`, the processor removes the `source_field` from the document. Default is `false`. |
+`description`  | Optional  | A brief description of the processor.  |
+`if` | Optional | A condition for running the processor. |
+`ignore_failure` | Optional | Specifies whether the processor continues execution even if it encounters errors. If set to `true`, failures are ignored. Default is `false`. |
```
[the remark this answers is not in the document]
Comment (gaobinlong) [reply _ingest-pipelines/processors/copy.md:44]: `ignore_failure` is a common parameter among all processors, we should change the description for all processors.

**Comment by natebower** [comment]
@gaobinlong Would you mind making these two commits?

**Pull request merged**